src/components/Book.jsx

```diff
@@ -0,0 +1,13 @@
+import PropTypes from 'prop-types';
+
+const Book = ({ title, author }) => (
+  <li>
+    <h2>{ title }</h2>
+    <p>{ author }</p>
+  </li>
+);
```

Comment:
- [ ] Good job so far 👏 However from the project requirements, here in the Book component there is supposed to be a `Remove` button even though you have not yet implemented its functionality. I would suggest that you add it there as just for static UI then in the next projects, we'll be implementing functionalities 😃